packages/expo/src/DangerZone.ts

```diff
@@ -3,12 +3,21 @@
  * up to date if you plan to use any of these.
  */
 
+import { Constants } from 'expo-constants';
```

Comment:
```suggestion
import Constants from 'expo-constants';
```
1. CI failed due to this,
2. I think `Constants` are exported as a default, not named export from `expo-constants`,
3. Please test these changes before pushing. 🙂